### PR TITLE
Intent to implement: `amp-ad` type: 'chargeads'

### PR DESCRIPTION
**Pull request by danielstockton**
Hi,

I'd like to request the addition of the chargeads type for the amp-ad component.
I signed the CLA on behalf of Chargeads.

Thanks!

### CODE REVIEW

File: ads/chargeads.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (dvoytenko) [inline ads/chargeads.js]: Could you please update this to 2016?

File: ads/chargeads.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+-->
+
+# Chargeads ⚡
+
+## Example
+
+### Basic
+
+```html
+  <amp-embed width=100 height=283
```
Comment (dvoytenko) [inline ads/chargeads.md]: I believe this should be `amp-ad`. Right?

Comment (dvoytenko) [inline ads/chargeads.md]: Or, at least, let's make this the same as examples below?

**Comment by dvoytenko** [comment]
@danielstockton Thanks! All looks good. Just a couple of minor comments.

The CLA hasn't come through yet. Can you make sure you sign with the same email you used for the commit?

**Comment by danielstockton** [comment]
It would not let me sign the CLA again with a new email so I added the email I signed with to my GitHub account. I hope this is OK.

**Comment by dvoytenko** [comment]
@danielstockton CLA looks good. But the PR needs to be rebased to remove merge conflicts. Could you please rebase as described in the https://help.github.com/articles/about-git-rebase/?

**Comment by dvoytenko** [comment]
@danielstockton Looks like the PR has been rebased. Merging...

**Pull request merged**